schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -10,21 +10,49 @@ vars:
     SOFTFAIL_BSC1063638: 1
     VALIDATE_ETC_HOSTS: 1
 schedule:
+    - "{{pre_boot_to_desktop}}"
     - boot/boot_to_desktop
     - console/prepare_test_data
     - console/consoletest_setup
     - console/hostname
-    - yast2_gui/yast2_control_center
+    - x11/yast2_snapper
+    - "{{yast2_modules}}"
     - x11/yast2_lan_restart
-    - yast2_gui/yast2_bootloader
     - yast2_gui/yast2_datetime
-    - yast2_gui/yast2_firewall
     - yast2_gui/yast2_hostnames
-    - yast2_gui/yast2_lang
-    - yast2_gui/yast2_network_settings
     - yast2_gui/yast2_software_management
     - yast2_gui/yast2_users
     - yast2_gui/yast2_security
     - yast2_gui/yast2_lan_restart_bridge
     - yast2_gui/yast2_lan_restart_vlan
     - yast2_gui/yast2_lan_restart_bond
+    - x11/yast2_snapper
```

Comment:
:) duplicate with L18

Reply:
Thank you very much for spending time on this , but this is WIP, I have made a bunch of experimental changes. I'll let you know when it's at review state.